CODEOWNERS

```diff
@@ -0,0 +1,122 @@
+/drivers/pca9685/                           @gschorcht
+/drivers/sht3x/                             @gschorcht
+/drivers/si70xx/                            @basilfx
+/drivers/sx127x/                            @aabadie @jia200x
+/drivers/ws281x/                            @maribu
+/drivers/xbee/                              @haukepetersen @miri64
+
+/pkg/cryptoauthlib/                         @Einhornhool @PeterKietzmann
+/pkg/lua/                                   @jcarrano
+/pkg/lwip/                                  @miri64
+/pkg/gecko_sdk/                             @basilfx
+/pkg/lora-serialization/                    @leandrolanzieri
+/pkg/micropython/                           @bergzand @kaspar030
+/pkg/openthread/                            @Hyungsin @jia200x
+/pkg/semtech-loramac/                       @aabadie @jia200x
+/pkg/tinydtls/                              @rfuentess @leandrolanzieri
+/pkg/u8g2/                                  @basilfx
+/pkg/ucglib/                                @basilfx
+/pkg/wakaama/                               @leandrolanzieri
+
+/sys/clif/                                  @leandrolanzieri
+/sys/event/                                 @kaspar030
+/sys/evtimer/                               @kaspar030
+/sys/fmt/                                   @kaspar030
+/sys/net/application_layer/gcoap/           @haukepetersen @kb2ma
+/sys/net/application_layer/nanocoap/        @kaspar030 @haukepetersen @kb2ma
+/sys/net/netif/                             @miri64 @jia200x
+/sys/net/gnrc/network_layer/                @miri64
+/sys/net/gnrc/transport_layer/tcp/          @brummer-simon @miri64
+/sys/net/gnrc/transport_layer/udp/          @miri64
+/sys/net/gnrc/link_layer/lorawan/           @jia200x
+/sys/net/gnrc/netif/                        @miri64 @jia200x
+/sys/net/gnrc/routing/rpl/                  @cgundogan @emmanuelsearch
+/sys/net/                                   @miri64 @haukepetersen @PeterKietzmann @cgundogan
+/sys/pm_layered/                            @roberthartung @kaspar030
+/sys/random/fortuna/                        @basilfx
+/sys/riotboot/                              @kaspar030 @fjmolinas
+/sys/suit                                   @kaspar030 @fjmolinas @bergzand
+/sys/tsrb/                                  @kaspar030
+/sys/usb/                                   @bergzand @dylad @aabadie
+/sys/xtimer/                                @kaspar030 @MichelRottleuthner
+
+/tests/                                     @miri64 @MrKevinWeiss @smlng @leandrolanzieri @aabadie @MichelRottleuthner @fjmolinas 
+/tests/cpu_efm32_features/                  @basilfx
+
+# KConfig maintainers will be notified about all KConfig changes
+Kconfig                                     @leandrolanzieri @jia200x @cgundogan
+Makefile.dep                                @aabadie @fjmolinas
+Makefile.features                           @aabadie @fjmolinas
+Makefile.include                            @aabadie @fjmolinas
```

Comment:
I'm worried of the implication of this, I don't feel confident and wont have the time to be responsible for the `build-system`. At least I think there should be much more people on this list.